### PR TITLE
[Testing] Added UI Test for manual test D13

### LINKED ISSUE #18703: [Testing] Create UI Test for manual test D13
### Description

Create UI Test for manual test D13.

```
<ScrollView>
    <VerticalStackLayout>
        <Label Text="1. The test fails if the text in the editor below is not in the top-left of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Start"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Start" />

        <Label Text="2. The test fails if the text in the editor below is not in the top-center of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Center"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Start" />

        <Label Text="3. The test fails if the text in the editor below is not in the top-right of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="End"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Start" />

        <Label Text="4. The test fails if the text in the editor below is not in the center-left of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Start"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Center" />

        <Label Text="5. The test fails if the text in the editor below is not in the center-center of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Center"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Center" />

        <Label Text="6. The test fails if the text in the editor below is not in the center-right of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="End"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="Center" />

        <Label Text="7. The test fails if the text in the editor below is not in the bottom-left of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Start"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="End" />

        <Label Text="8. The test fails if the text in the editor below is not in the bottom-center of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="Center"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="End" />

        <Label Text="9. The test fails if the text in the editor below is not in the bottom-right of the editor." />
        <Editor BackgroundColor="LightBlue"
                HeightRequest="100"
                HorizontalTextAlignment="End"
                IsReadOnly="True"
                Text="testing"
                TextColor="Black"
                VerticalTextAlignment="End" />

        <Label Text="10. The test fails if the text in the editor below is not in the center-center of the editor." />
        <Grid HeightRequest="100">
            <Editor BackgroundColor="LightBlue"
                    HorizontalTextAlignment="Center"
                    IsReadOnly="True"
                    Text="testing"
                    TextColor="Black"
                    VerticalTextAlignment="Center" />
        </Grid>
    </VerticalStackLayout>
</ScrollView>
```

### Steps to Reproduce

_No response_

### Link to public reproduction project repository

_No response_

### Version with bug

8.0.0-rc.2.9511

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

_No response_

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Added UI Test for manual test D13.

### Issues Fixed

Fixes #18703

### CODE REVIEW

File: src/Controls/tests/UITests/snapshots/ios/EditorTextAlignmentWorks.png
Comment (PureWeen) [inline src/Controls/tests/UITests/snapshots/ios/EditorTextAlignmentWorks.png:1]: Can't see the bottom editor field on android or iOS. I wonder if we could use device tests to locate the text somehow? Or maybe these editors fields can be side by side?

**Review by PureWeen — CHANGES_REQUESTED** [review]

File: src/Controls/tests/UITests/snapshots/ios/EditorTextAlignmentWorks.png
Comment (jsuarezruiz) [reply src/Controls/tests/UITests/snapshots/ios/EditorTextAlignmentWorks.png:1]: Applied changes to the sample to capture all the cases in the snapshot.
Screenshot_1701269609

Comment (jsuarezruiz) [reply src/Controls/tests/UITests/snapshots/ios/EditorTextAlignmentWorks.png:1]: NOTE: Waiting the build to generate the new snapshots on CI.

**Review by hartez — DISMISSED** [review]

**Comment by jfversluis** [comment]
We're focussing on other areas of the testing right now. Lets close this in the meantime and revisit later.

**Pull request closed without merging**